src/registry_schemas/schemas/agm_location_change.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/agm_location_change",
+    "type": "object",
+    "definitions": {},
+    "title": "AGM Location Change Information Schema",
+    "required": [
+        "agmLocationChange"
+    ],
+    "properties": {
+        "agmLocationChange": {
+            "type": "object",
+            "required": [
+                "year",
+                "newAgmLocation"
+            ],
+            "properties": {
+                "year": {
+                    "type": "string",
+                    "description": "Year of the AGM, Must be on or after incorporation year and cannot be future year."
+                },
+                "newAgmLocation": {
```

Comment:
Could this use the Address Schema?

Reply:
Actually, yes I think we can.  

Maybe something along the lines of this.  Filing schema does something similar too -> https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/filing.json#L7-L23

Does something like this work for you, @thorwolpert ?

```json
"newAgmLocation": {
      "allOf": [
        { "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address" },
        {
          "properties": {
				"additionalLocationDetails": {
                 	"type": ["string", "null"],
                    "maxLength": 2000 }
          },
        }]
}
```

Reply:
I think it will work

Reply:
Hi @thorwolpert 

I updated the `newAgmLocation` to use the Address Schema.
Please review it again.